shared-ui/src/commonMain/kotlin/co.touchlab.droidcon/util/NavigationController.kt

```diff
@@ -130,20 +128,20 @@ class NavigationController : BaseViewModel() {
     }
 
     @Composable
-    private fun <T: Any> PushedStackItem(item: NavigationStackItem.Push<T>, itemModifier: Modifier) {
+    private fun <T : Any> PushedStackItem(item: NavigationStackItem.Push<T>, itemModifier: Modifier) {
         println("$item")
         val itemValue by item.item.observeAsState()
 
         itemValue?.let {
-            Surface {
+            Surface(modifier = itemModifier) {
                 item.content(it)
             }
         }
     }
 
     @Composable
     internal fun <T : Any> Pushed(item: MutableObservableProperty<T?>, content: @Composable (T) -> Unit) {
-        val refTracking = remember {
```

Comment:
the name refTracking makes me wonder if it was intentionally assigning to a variable even though it wasn't used. I still imagine it isn't needed, but maybe double check

Reply:
I tested the navigation and it works correctly, I think.